website/source/docs/commands/force-leave.html.markdown.erb

```diff
@@ -10,20 +10,24 @@ description: |-
 
 Command: `consul force-leave`
 
-The `force-leave` command forces a member of a Consul cluster to enter the
-"left" state. If the member is still actually alive, it will
-eventually rejoin the cluster. The true purpose of this method is to force
-remove "failed" nodes.
+The `force-leave` command forces a departed member of a Consul cluster to enter the
+"left" state. The purpose of this method is to force-remove nodes that have failed or 
```

Comment:
```suggestion
"left" state. The purpose of this method is to force-remove a node that has failed or 
```
Since to command needs to be run per failed node, I recommend changing this to be singular.